helixbass/tree-sitter-lint
# Use rule languages

In this PR:
- wire up basic support for the languages declared by rules to drive which language(s) to perform linting for

To test:
If you `cargo run -- --force-rebuild` you should see lint errors for both Rust and TOML

Based on `use-get-captures`
